### PR TITLE
less confusing readme for ansible newbies #74

### PR DESCRIPTION
**Pull request by pdurbin**
Relates to #74

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The preparation lies in the group_var options (usernames/passwords, whether to i
 ### Usage:
 	$ git clone https://github.com/IQSS/dataverse-ansible.git dataverse
 	$ export ANSIBLE_ROLES_PATH=.
-	$ ansible-playbook -i <inventory file> [-u <user>] [-b] [-K] -e @<group_vars_file> [-v] dataverse.pb
```
Comment (pdurbin) [inline README.md]: Above `group_vars_file` is still mentioned. Maybe we should update it?

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ The preparation lies in the group_var options (usernames/passwords, whether to i
 ### Usage:
 	$ git clone https://github.com/IQSS/dataverse-ansible.git dataverse
```
Comment (pdurbin) [inline README.md]: Should we change `$` to `#` and mention above to run these commands as root?

Comment (donsizemore) [reply README.md]: to me `$` means prompt; `#` means comment.

`-b` and `-K` were in there but may be expounded upon?

**Comment by pdurbin** [comment]
Closing in favor of #78

**Pull request closed without merging**